### ISSUE TITLE
WHAT, HOW, WHY Install OpenWhisk on the main page.

### ISSUE THREAD

**Issue by csantanapr** [apache/incubator-openwhisk-website#255]
We are getting multiple negative feedback that when people the website they don't know how to install OpenWhisk.

Kubernetes has a good website we could follow same pattern and UX
https://kubernetes.io/docs/home/?path=users&persona=app-developer&level=foundational

There might be different levels of users.
We should have simple and clear guidance for the type of User
I AM...
- Application Developer
  - I just want to run code in a hosted offering
  - I just want to locally test, debug code that I'm about to run in a hosted offering (i.e. local-nodejs, local-java, serverless framework local run, vscode debugging, Shell UI debugger)
- Cluster Operator
  - I want to deploy opewhisk cluster locally in my laptop
    - Use docker-compose (Linux and Mac only)
    - Use wskdev fresh (ansible Linux and Mac only)
    - Use Vagrant VM (Windows, for linux and mac recommend use docker-compose or wskdev)
    - Use minikube 
    - Use minishift
    - Use mesos (local)
   - I want to deploy openwhisk remote cluster
     - Setup a couple of hosted VMs (Ubuntu and ssh) and use ansible
     - Use kubernetes from a provider (IBM Cloud, etc..)
     - Use openshift from a provider (Redhat, etc..)
     - Use mesos from a provider provider (?)

There is an issue trying to consolidate the information in to the main core repo:
https://github.com/apache/incubator-openwhisk/issues/3498

cc @akrabat @mrutkows @rabbah @jthomas @ddragosd @tysonnorris @dgrove-oss

**Comment by krook** [comment]
There were three roles we identified for the first version of the site. We called them three things Developer, Contributor, and Package Creator. There's also a fourth, which we, unfortunately, left out but which you included @csantanapr.

- Developer (In your list as Application Developer, the end user, person with the serverless point of view).
- Cluster Operator (In your list, person who provides the FaaS platform to the End User/Developer).
- Contributor (The person who wants to understand and contribute to the open source project).
- Package Creator (The person who wants to provide integration, could be a third party like Twilio, or someone who wants to integrate with MySQL).
